beetsplug/chroma.py

```diff
@@ -182,11 +183,15 @@ def __init__(self):
             self.register_listener("import_task_start", self.fingerprint_task)
         self.register_listener("import_task_apply", apply_acoustid_metadata)
 
+    @cached_property
```

Comment:
I think it could be worth it implementing a singleton wrapper for all plugins. I dont think we ever need multiple instances of a plugin.

I guess out of scope for this PR tho.

Reply:
Good idea. I think this could be as simple as adding a function `beets.plugins.get(name: str)` which gets the relevant plugin from the `_instances` mapping.

Reply:
Not going to add it here because imo `chroma` should work fine even though users have disabled `musicbrainz` plugin.